app-portage/prefix-toolkit/prefix-toolkit-10-r1.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DESCRIPTION="Utilities for users of Gentoo Prefix"
-HOMEPAGE="https://prefix.gentoo.org/"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:Prefix"
```

Comment:
what's wrong with the prefix.gentoo.org url here?

Reply:
there's nothing particularly wrong with prefix.gentoo.org.
I made the change to save an redirect to to `https://wiki.gentoo.org/wiki/Project:Prefix`
It's only a minor change, but i think it a nice to have change.